pkgs/tools/misc/godu/default.nix

```
@@ -0,0 +1,24 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  pname = "godu";
+  version = "1.3.0";
+  rev = "v${version}";
+
+  goPackagePath = "github.com/viktomas/godu";
+
+  src = fetchFromGitHub {
+    owner = "viktomas";
+    repo = "godu";
+    inherit rev;
+    sha256 = "1fp8iq4x0qiswksznnd6qh7c6g5pwglzz6ga11a7vgic0201wsvb";
+  };
+
+  goDeps = ./deps.nix;
+
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/viktomas/godu";
+    description = "Simple golang utility helping to discover large files/folders.";
```

Description should not end with a period.

```suggestion
    description = "Simple golang utility helping to discover large files/folders";
```